src/helpers/i18n/fr/translation.json

```diff
@@ -86,50 +86,49 @@
     }
   },
   "DATA_SHARING": {
-    "SHARED_DATA": "",
-    "BOARDINGS_FROM": "",
-    "BOARDINGS_TO": "",
-    "WHO_CAN_ACCESS_DATA": "",
-    "SHARING_DATA_WITH": "",
-    "FOLLOWING_AGENCIES": "",
-    "SHARING_DATA_WITH_ZERO_AGENCIES": "",
-    "AGENCY_ADMINS": "",
+    "SHARED_DATA": "Données Partagées",
+    "BOARDINGS_FROM": "BOARDINGS FROM",
+    "BOARDINGS_TO": "À",
+    "WHO_CAN_ACCESS_DATA": "Qui peut accéder aux données",
+    "SHARING_DATA_WITH": "{{agency}} informe {{sharingAgency}}",
```

Comment:
This should be the translation of "Agencies sharing data with {{agency}}". (as above)